doc/source/installation.rst

```diff
@@ -69,6 +69,7 @@ For example, here are the Ray 2.0.0.dev0 wheels for Python 3.5, MacOS for commit
 
 Install Ray With Maven
 ----------------------
+Before install it with Maven, you should install Ray by `pip install -U ray` .
```

Comment:
```suggestion
Before installing Ray Java with Maven, you should install Ray Python with `pip install -U ray` . Note that the versions of Ray Java and Ray Python must match.
```

Reply:
Please also mention that nightly Ray Python wheels are required if a user wants to install the Ray Java snapshot version.